### PR TITLE
{chem}[intel/2018b] AtomPAW v4.1.0.5

### PR DESCRIPTION
**Pull request by jmbeuken**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jmbeuken** [comment]
Thi is the last version usable with ABINIT 8.10.x : there is a problem with the AtomPAW 4.1.0.6 !
When this PR will be merged, I should modify ABINIT-8.10.3-intel-2018b.eb...

File: easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+##
+# This version of AtomPAW is lastest version to be used with ABINIT 8.10.x
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb]: if it's a problem with `AtomPAW` 4.1.0.6, wouldn't it be better to add this comment in that easyconfig, or on the `ABINIT` easyconfig?

If a later version of `AtomPAW` can be used with `ABINIT`, this comment will be outdated...

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/feaffda1cad825f4a6dc7b7b16a06445 for a full test report.

File: easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb (hunk 1 above)
Comment (jmbeuken) [reply easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb]: Thank you for this comment....
I was indeed very annoyed... :-(

To clarify the situation:

1. AtompPAW version 4.1.0.6 has no intrinsic problems
2. ABINIT version 8.10.3 is the latest in the 8.10 series
3. The future production version of ABINIT will be a version 9.0 which is compatible with AtomPAW 4.1.0.6
4. We could even remove the sentence in the AtomPAW 4.1.0.5 config file : 

`# This version of AtomPAW is lastest version to be used with ABINIT 8.10.x`

Comment (migueldiascosta) [reply easybuild/easyconfigs/a/AtomPAW/AtomPAW-4.1.0.5-intel-2018b.eb]: ok, then the sentence seems correct, although I'd say it's more important to state that in the `ABINIT ` easyconfig than here

**Comment by migueldiascosta** [comment]
Going in, thanks @jmbeuken!

**Pull request merged**